_data/sessions.yml

```diff
@@ -1,368 +1,713 @@
+      This talk explores a different way of building networks that does not depend on distributed consensus. By making a small number of tradeoffs, it is possible to build a network that can scale to 10^3x the speed in inserting new data and distributing it. Aether is one such attempt, and both it and the underlying protocol is open source. This talk will focus on how it ended up coming to be, why, and how it works, and what you can build with the underlying protocol."
+    descriptionLink: "https://github.com/ournetworks/2018-submissions/issues/25"
+    sessionType: "talk"
+    date: 2018-07-14
+    length: 0.5
+    youtube: "A9zL2csO7t8"
+    ipfs720: "QmRvHxC16K1hqeWdmtGNQQ6jnogq9pfmbVo4VSZqeET8dE"
+    ipfs1080: "QmPk33q8z8GHhJoDKXGDGn5CWFu61TzTprgupfGe4VNwho"
+
+  - sessionID: 18991
+    title: "Opening remarks"
+    descriptionLink: "https://github.com/ournetworks/2018-submissions/issues/"
+    sessionType: "orga"
+    date: 2018-07-14
+    length: 0.5
+    youtube: "0X95Nybu2g8"
+    ipfs720: "QmcCXAbjDwnkhzL8qfrbJLH6DqEbrkBvaF8AiXsdXNLiDL"
+    ipfs1080: "QmZJ5ZbJTfhQXM57PSWYy1VsHRDfL5AckjXy6W7RmP89AX"
+
+  - sessionID: 18992
+    title: "Closing remarks"
+    sessionType: "orga"
+    date: 2018-07-15
+    length: 0.5
+    youtube: "hwjeNX-9K58"
+    ipfs720: "QmRmJip9agFXbckr8ZPFb4Fdwn54Xi9iyWh1NdT1D7cmtS"
+    ipfs1080: "QmRVhwmQEGdxeLNUq41exH8GJTXY4ZPV45Ktto8VxJRhi7"
+
+  - sessionID: 18993
+    title: "Lightning Talks"
+    sessionType: "lightning"
+    date: 2018-07-15
+    length: 1.0
+
+  - sessionID: 18994
+    title: "Sprint Pitches"
+    sessionType: "sprint-pitch"
+    date: 2018-07-15
+    length: 0.5
+
+  - sessionID: 18997
+    title: "Keynote: Nasma Ahmed, Navigating digital futures"
+    sessionType: "keynote"
+    date: 2018-07-14
+    length: 1
+
+2019:
+  - sessionID: 19990
+    title: "Registration"
+    sessionType: "orga"
+
+  - sessionID: 19991
+    title: "Coffee break"
+    sessionType: "orga"
+    length: 0.25
+
+  - sessionID: 19992
+    title: "Lunch"
+    sessionType: "orga"
+    
+  - sessionID: 19993
+    title: "Kickoff party"
+    sessionType: "orga"    
+  
+  - sessionID: 19994
+    title: "Keynote"
+    sessionType: "keynote"
+    date: 2019-09-20
+    length: 1.5
+  
+  - sessionID: 19997
+    title: "Lightning talks"
+    sessionType: "lightning"
+    date: 2019-09-22
+    length: 1.0
+  
+  - sessionID: 19998
+    title: "Opening remarks"
+    sessionType: "orga"
+    date: 2019-09-21
+    length: 0.5
+  
+  - sessionID: 19999
+    title: "Closing remarks"
+    sessionType: "orga"
+    date: 2019-09-22
+    length: 0.5
+
+  - sessionID: 19003
+    title: "DHT, BFT, OMG: Know your building blocks for decentralized applications (2019 edition)"
+    descriptionLink: "https://github.com/ournetworks/2019-submissions/issues/3"
+    presenterID:
+      - kirill
+    description: > 
```

Comment:
@dcwalk Noticing that double-quotes are showing as the first character on session captions and it looks like this is causing it. Is there a case for descriptions to use `>`?

Reply:
I think I mucked this up -- I was having issues with description length in just double quotes, so I switched to multiline, cleaned up in https://github.com/ournetworks/ournetworks.ca/pull/118/commits/5a7ee89bc466181a5d905e0eadee49d84cd4591f